tests/wicked/config_files.pm

```diff
@@ -29,18 +31,30 @@ sub reset_network {
 sub run {
     my ($self) = @_;
     $iface = script_output('echo $(ls /sys/class/net/ | grep -v lo | head -1)');
-    $self->get_from_data('wicked/ifcfg-eth0',           "/data/ifcfg-$iface");
-    $self->get_from_data('wicked/static-addresses.xml', "/data/static-addresses.xml");
-    $self->get_from_data('wicked/ifbind.sh',            '/bin/ifbind.sh', executable => 1);
-    assert_script_run("sed -i 's/xxx/$iface/g' /data/static-addresses.xml");
+    $self->get_from_data('wicked/static_address/ifcfg-eth0',             "/data/static_address/ifcfg-$iface");
```

Comment:
I would put these paths to the variable, so could adjust them in one place if requires. Otherwise LGTM

Reply:
make sense , but not sure how to do this to not over complicate things. 
basically each function has two strings 1st - path to a file to download , 2nd - target place+name. 
creating two arrays I think just confuse user. prefer to see  
`$self->get_from_data('wicked/static_address/ifcfg-eth0',             "/data/static_address/ifcfg-$iface");`
instead of 
`$self->get_from_data(file_to_download[1], place_to_save[1]);` which will not bring a lot info even with proper array names.
So if you can suggest some certain solution which will achieve what you suggested without loosing visibility what happening I am open to follow

Reply:
We could put them in just 2 variables, but let's consider it for future